src/e2e-test/java/io/cdap/plugin/servicenowsink/actions/ServiceNowSinkPropertiesPageActions.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.servicenowsink.actions;
+
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.servicenow.apiclient.ServiceNowTableAPIClientImpl;
+import io.cdap.plugin.servicenow.locators.ServiceNowPropertiesPage;
+import io.cdap.plugin.servicenow.source.ServiceNowSourceConfig;
+import io.cdap.plugin.servicenow.util.ServiceNowConstants;
+import io.cdap.plugin.tests.hooks.TestSetupHooks;
+import org.apache.oltu.oauth2.common.exception.OAuthProblemException;
+import org.apache.oltu.oauth2.common.exception.OAuthSystemException;
+import org.junit.Assert;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+/**
+ * ServiceNow sink - Properties page - Actions.
+ */
+
+public class ServiceNowSinkPropertiesPageActions {
+  private static final int DEFAULT_SCALE = 2;
+  public static ServiceNowSourceConfig config;
+  public static Map<String, String> responseFromServiceNowTable;
```

Comment:
why do these need to be public?

Reply:
Not needed as public, Changed to private.(https://github.com/data-integrations/servicenow-plugins/pull/35/commits/04c5fcb85a4ffccdcfd37580d88f6ae267c0f49e#diff-cc3534f5337d36a161fd6bddf24a37827ae2f6ff149c7f26a36a0346f846b118R118)